src/components/common/ColonyActions/ActionsPage/TransactionStatus/TransactionStatus.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { defineMessages, FormattedMessage } from 'react-intl';
+
+import { Tooltip } from '~shared/Popover';
+import { getMainClasses } from '~utils/css';
+
+import { Status } from '../staticMaps';
```

Comment:
I think this naming lacks the word "transaction" in it, so that it's clear straightaway about which status we're talking

Reply:
Renamed to TransactionStatuses to avoid conflict with the component